### ISSUE TITLE
Create branch which will auto-publish new version

### ISSUE THREAD

**Issue by WikKam** [Wokstym/react-vis-graph-wrapper#12]
Apart from master, we should create `publish` branch and configure the pipeline such that its changes will be auto-published

**Comment by ZachHaber** [comment]
That sounds useful!
Along with that I'd propose that we use conventional changelog and auto semantic versioning so that we can get automated changelog generation.
It's a little annoying to work with, but it encourages trying to keep commits self contained.

**Comment by Wokstym** [comment]
@ZachHaber This is lib that you meant? I can get started on that 
https://github.com/conventional-changelog/conventional-changelog

**Comment by ZachHaber** [comment]
Yeah, though apparently we should use the higher level https://github.com/conventional-changelog/standard-version library for it.

**Comment by Wokstym** [comment]
So i propose I will add `standard-version` dependency and when we decide it is time to publish new tag we will create pr adding pipeline on a new production branch.
Let's start using new commit convention
